### DISCUSSION TITLE
inventree Set debug to False then can't see dashboard

### DISCUSSION THREAD

**Discussion by pityAB** [inventree/InvenTree#8124] [Q&A] [answered]
sorry in advance this is my first experience installing inventree with bare metal on windows. and I haven't had time to read the entire discussion on this forum. I'm just confused after I set debug to false then I run the server, then I log in and succeed, but the initial display of the main menu dashboard is gone and all part/category data is not displayed. please enlighten me. and thank you in advance
![indexInv](https://github.com/user-attachments/assets/4a90c52a-4298-4a8a-ae55-006bc8c0b50e)
then
![partInv](https://github.com/user-attachments/assets/c61ce1e1-1627-4055-9b11-2aa61cbbff27)

**Comment by SchrodingersGat** [comment, marked as answer]
This is clearly laid out in the documentation - https://docs.inventree.org/en/stable/start/intro/#potential-issues

When you disable "debug" mode, the backend server does not host static files. You'll need a separate fileserver for that.

https://docs.inventree.org/en/stable/start/serving_files/

This is a bit of a pain to setup in bare metal  but you can read the docs here -https://docs.inventree.org/en/stable/start/bare_prod/

Alternatively, you can use our [docker setup](https://docs.inventree.org/en/stable/start/docker_install/) or [installer](https://docs.inventree.org/en/stable/start/installer/)

**Reply by pityAB** [reply]
thanks for the instructions. I can try further and those alternatives first
